Libraries/lwIP/include/Maxim/arch/sys_arch.h

```diff
@@ -1,35 +1,95 @@
-/******************************************************************************
+/*
+ * Copyright (c) 2017 Simon Goldschmidt
+ * All rights reserved.
  *
- * Copyright (C) 2022-2023 Maxim Integrated Products, Inc. (now owned by 
- * Analog Devices, Inc.),
- * Copyright (C) 2023-2024 Analog Devices, Inc.
```

Comment:
@demiromerb copyright changed on this file - is this correct?

Reply:
Yes, that is correct. The old file was just a placeholder for some variables. New file is fetched directly from the copyright holders work. Nothing changed by us so,  as far as I know we cannot add our copyright on top of that.